src/test/java/space/obminyashka/items_exchange/service/FavoriteAdvertisementDBTest.java

```diff
@@ -0,0 +1,41 @@
+package space.obminyashka.items_exchange.service;
+
+import com.github.database.rider.core.api.dataset.DataSet;
+import com.github.database.rider.core.api.dataset.ExpectedDataSet;
+import com.github.database.rider.junit5.api.DBRider;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import space.obminyashka.items_exchange.dao.AdvertisementRepository;
+import space.obminyashka.items_exchange.dao.UserRepository;
+
+import java.util.List;
+import java.util.UUID;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+@SpringBootTest
+@DBRider
+class FavoriteAdvertisementDBTest {
+    @Autowired
+    private UserRepository userRepository;
+    @Autowired
```

Comment:
With JUnit 5 you can use Lombok's RequiredAllArgsConstructor and to inject fields without Autowired